docs/usage/node-readiness.md

```diff
@@ -19,14 +19,26 @@ The controller removes the taint once all node-critical Pods are ready (determin
 The `Node` controller considers all `DaemonSets` and `Pods` with the label `node.gardener.cloud/critical-component=true` as node-critical.
 If there are `DaemonSets` that contain the `node.gardener.cloud/critical-component=true` label in their metadata and in their Pod template, the `Node` controller waits for corresponding daemon Pods to be scheduled and to get ready before removing the taint.
 
+Additionally, the `Node` controller checks for the readiness of `csi-driver-node` components if a respective Pod indicates that it uses such a driver.
+This is achieved through a well-defined annotation prefix (`node.gardener.cloud/wait-for-csi-node-`).
+E. g. for an Openstack Cinder CSI Driver, the `csi-driver-node` Pod is annotated with `node.gardener.cloud/wait-for-csi-node-cinder=cinder.csi.openstack.org`.
+The annotation key's suffix is arbitrarily chosen (in this case `cinder`) and the annotation value needs to match the actual driver name as specified in the `CSINode` object.
```

Comment:
Can you extend this with the motivation for this "prefix" approach? From the issue, I read:

> a key prefix is used here to allow a singlecsi-driver-node pod to specify multiple CSI drivers that are registered by it (using the value for a fixed key would allow specifying only one CSI driver)